sources/code/common/main.ts

```diff
@@ -124,6 +124,16 @@ let startHidden = false;
  */
 let screenShareAudio = false;
 
+/* eslint-disable */
+try {
+  const lib = require("node-pipewire");
+  lib.createPwThread(argv.values.verbose === true);
+  screenShareAudio = true;
+} catch(e) {
+  console.warn(e);
+}
+/* eslint-enable */
```

Comment:
You should avoid using `eslint-disable` for all of the rules and lines there and explain why you disable the linter. I won't allow *bypassing* the problems – you should resolve the linter rules or explain why you didn't for each.